### PR TITLE
Update sites.js

### PR DESCRIPTION
**Pull request by serocell**

### CODE REVIEW

**Comment by kodedninja** [comment]
There's a missing comma from the line before yours and there's one at the end of your site's line. Please add the first one and remove the second.

**Comment by kodedninja** [comment]
@neauoire does this count as a single page website?

**Comment by neauoire** [comment]
Yeah, it's not a great candidate for the webring. 
But let's give them a week to improve their site.

File: scripts/sites.js [hunk 1]
```diff
@@ -90,6 +90,6 @@ const sites = [
   { url: 'https://amorris.ca', title: 'amorris', author: 'amorris', type: 'blog', feed: 'https://feed.amorris.ca/hallway.txt' },
   { url: 'http://www.miha-co.ca', title: 'miha-co', type: 'Portfolio' },
   { url: 'https://notes.stuartpb.com/', title: 'notes.stuartpb.com', type: 'wiki', author: 'stuartpb', contact: 's@stuartpb.com', feed: 'https://twtxt.stuartpb.com/xxiivv.txt' }
-  { url: 'http://xxiii.co' },
+  { url: 'http://xxiii.co' title: 'xxiii', type: 'portfolio', author: 'serocell', contact: 'psignal@s900.net', rss: 'http://serocell.com/feeds/serocell.xml', feed: 'http://xxiii.co/twtxt.txt' },
```
Comment (neauoire) [inline scripts/sites.js]: Missing comma, please test before making PRs.

Comment (cblgh) [reply scripts/sites.js]: is this why the webring is down?

Comment (neauoire) [reply scripts/sites.js]: yeah I shouldn't have merged @serocell's PR, I'll revert :)

Comment (neauoire) [reply scripts/sites.js]: Fixed, sorry about that :)

**Pull request closed without merging**